Adyen.Test/BaseTest.cs

```diff
@@ -341,13 +355,15 @@ protected Client CreateMockTestClientApiKeyBasedRequest(string fileName)
         /// <returns>IClient implementation</returns>
         protected Client CreateMockTestClientPosCloudApiRequest(string fileName)
         {
-            var config = new Config {Endpoint =ClientConfig.CloudApiEndPointTest};
+            var config = new Config {Endpoint = ClientConfig.CloudApiEndPointTest};
             var mockPath = GetMockFilePath(fileName);
             var response = MockFileToString(mockPath);
             //Create a mock interface
             var clientInterfaceMock = new Mock<IClient>();
             clientInterfaceMock.Setup(x => x.Request(It.IsAny<string>(),
-                It.IsAny<string>(), config,It.IsAny<bool>(), It.IsAny<RequestOptions>())).Returns(response);
+                It.IsAny<string>(), config, It.IsAny<bool>(), It.IsAny<RequestOptions>())).Returns(response);
+            clientInterfaceMock.Setup(x => x.RequestAsync(It.IsAny<string>(),

```

Comment:
In the same method the clientInterfaceMock is Setup two times. In this way the CreateMockTestClientPosCloudApiRequest instead of Request is using RequestAsync. Can you create another method for async which uses the RequestAsync